### PR TITLE
colors.ts: ensure correct names are provided

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Fixes https://github.com/NWACus/avy/issues/416

### CODE REVIEW

File: components/core/View.test.tsx [hunk 1]
```diff
@@ -25,39 +26,39 @@ describe('View', () => {
   });
```
Comment (stevekuznetsov) [inline components/core/View.test.tsx:26]: The test above this shows an edge-case for how we have these factored right now - colors not given as top-level attributes don't get put through `colorLookup`. Getting stuff like this ironed out is likely harder than just using Tamagui.

**Review by floatplane — APPROVED** [review]
This seems like a good start! How many things break if you change this line:

https://github.com/NWACus/avy/blob/37c0dcd8642dfa2e9eea8f6a47e1289965734a07/components/core/View.tsx#L9

to:
```js
readonly backgroundColor?: ColorName,
```

File: components/core/View.test.tsx (hunk 1 above)
Comment (stevekuznetsov) [reply components/core/View.test.tsx:26]: Yes, I think the way they do Styled components does entirely make this error case impossible.
